packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -57,6 +102,32 @@ export default function useFeedSettings(): FeedSettingsReturnType {
     }, 100);
   }, [tagsCategories, feedSettings, avoidRefresh]);
 
+  useEffect(() => {
+    const isFeedQueryEmpty = isObjectEmpty(feedQuery);
+    const isFeedSettingsEmpty = isObjectEmpty(feedSettings);
+    if (
+      !isFeedSettingsEmpty ||
+      !loadedUserFromCache ||
+      isFeedQueryEmpty ||
+      !shouldShowMyFeed
+    ) {
+      return;
+    }
+
+    const localFeedSettings = getLocalFeedSettings();
+    client.setQueryData<AllTagCategoriesData>(filtersKey, (current) => ({
+      ...current,
+      feedSettings: { ...localFeedSettings },
+    }));
+  }, [feedQuery, feedSettings, loadedUserFromCache, user]);
+
+  useEffect(() => {
+    if (!hasClearedCache && loadedUserFromCache && user) {
+      hasClearedCache = true;
+      storage.removeItem(LOCAL_FEED_SETTINGS_KEY);
+    }
+  }, [loadedUserFromCache, user]);
```

Comment:
What's that part?

Reply:
This is to set the `feedSettings` initial values for anonymous users and part of the experiment since the property is undefined not `loggedIn`